charts/bindplane/values.yaml

```
@@ -105,6 +105,13 @@ image:
   # -- Image tag to use. Defaults to the version defined in the Chart's release.
   tag: ""
 
+transform_agent_image:
+  # -- Transform Agent Image name to be used. Defaults to `ghcr.io/observiq/bindplane-transform-agent`.
+  name: ""
+  # Overrides the image tag whose default is {{ .Chart.AppVersion }}
+  # -- Transform Agent Image tag to use. Defaults to latest.
+  tag: ""
```

Thinking about the future. I think we can stick all transform agent options in here. Things like resource request / limits, tls, etc.
```suggestion
transform_agent:
  image:
    # -- Transform Agent Image name to be used. Defaults to `ghcr.io/observiq/bindplane-transform-agent`.
    name: ""
    # Overrides the image tag whose default is {{ .Chart.AppVersion }}
    # -- Transform Agent Image tag to use. Defaults to latest.
    tag: ""
```